packages/wpcom.js/History.md

```diff
@@ -1,29 +1,31 @@
-# 6.0.0 / 2020-XX-XX
+## History
```

Comment:
Should this be:

```suggestion
# History
```

Reply:
Good catch! I get too carried away with the replacement 👍